statistics/handle/cache/internal/inner.go

```diff
@@ -21,29 +21,33 @@ import (
 // StatsCacheInner is the interface to manage the statsCache, it can be implemented by map, lru cache or other structures.
 type StatsCacheInner interface {
 	// GetByQuery retrieves cache triggered by a query. Usually used in LRU.
+	// TODO: merge this method with Get.
 	GetByQuery(int64) (*statistics.Table, bool)
 	// Get gets the cache.
 	Get(int64) (*statistics.Table, bool)
 	// PutByQuery puts a cache triggered by a query. Usually used in LRU.
+	// TODO: merge this method with Put.
 	PutByQuery(int64, *statistics.Table)
 	// Put puts a cache.
 	Put(int64, *statistics.Table)
 	// Del deletes a cache.
 	Del(int64)
 	// Cost returns the memory usage of the cache.
 	Cost() int64
-	// Keys returns the keys of the cache.
-	Keys() []int64
 	// Values returns the values of the cache.
 	Values() []*statistics.Table
-	Map() map[int64]*statistics.Table
```

Comment:
Remove 2 unnecessary methods to make this interface easier to maintain and understand.